utils/python/CIME/XML/grids.py

```diff
@@ -27,6 +27,19 @@ def get_grid_info(self, name, compset):
         """
         nodes = self.get_nodes("grid")
         gridinfo = {}
+        atmnlev = None
```

Comment:
Grid specification can optionally include vertical grid info for clm and cam.  Add support for that feature here.